### PR TITLE
[IMPROVE] Updated links in readme

### LINKED ISSUE #24027: Some broken URLs in Readme
### Description:
Some links in Readme need updating like
- Updating community open call link to the latest one i.e., December
- The API link redirects to https://developer.rocket.chat/ whereas it should be redirecting to API docs
- Contact us link is also not right

### Steps to reproduce:

1. Go to Readme
2. Click on the links that i stated above

### Server Setup Information:

- Version of Rocket.Chat Server: develop
- Operating System: linux

### Client Setup Information

- Desktop App or Browser Version: browser
- Operating System: linux

### PR DESCRIPTION
**Pull request by aswinidev**
## Issue(s)

Fixes #24027
## Steps to test or reproduce

Mentioned in the issue

### CODE REVIEW

**Comment by dudanogueira** [comment]
Hi!

Can you change the youtube link for the Open Community Call to:
https://www.youtube.com/playlist?list=PLee3gqXJQrFVaxryc0OKTKc92yqQX9U-5

this will point to the playlist instead of the latest video.

Also, the link https://rocket.chat/newsletter is redirecting to our blog instead of a proper newsletter page.
I have asked internally if we should keep this and change the link from here.

Thanks Aswini!

**Comment by aswinidev** [comment]
I made the required changes.
Do let me know about the newsletter link. Actually the blogs page has the newsletter subscription thing. I think that's why this is redirecting to the blogs page
Screenshot from 2021-12-27 18-21-06

**Comment by aswinidev** [comment]
@dudanogueira any updates on this? Please review.

File: README.md [hunk 1]
```diff
@@ -68,6 +68,6 @@ We're hiring developers, support people, and product managers all the time. Plea
 * [Youtube](https://www.youtube.com/channel/UCin9nv7mUjoqrRiwrzS5UVQ)
 * [Email Newsletter](https://rocket.chat/newsletter)
 
-Any other questions, reach out to us at [contact@rocket.chat](contact@rocket.chat). We’d happy to help!
+Any other questions, reach out to us at [contact@rocket.chat](https://rocket.chat/contact). We’d happy to help!
```
Comment (debdutdeb) [inline README.md:71]:
```suggestion
Any other questions, reach out to us at [our website](https://rocket.chat/contact) or you can email us directly at [contact@rocket.chat](mailto:contact@rocket.chat). We’d happy to help!
```

**Review by debdutdeb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aswinidev) [reply README.md]: I made the requested changes :+1: 
Please review!

**Comment by aswinidev** [comment]
@debdutdeb Please review! :smile:

File: README.md [hunk 2]
```diff
@@ -68,6 +68,6 @@ We're hiring developers, support people, and product managers all the time. Plea
 * [Youtube](https://www.youtube.com/channel/UCin9nv7mUjoqrRiwrzS5UVQ)
 * [Email Newsletter](https://rocket.chat/newsletter)
 
-Any other questions, reach out to us at [contact@rocket.chat](contact@rocket.chat). We’d happy to help!
+Any other questions, reach out to us at [our website](https://rocket.chat/contact) or you can email us directly at [contact@rocket.chat](mailto:contact@rocket.chat). We’d happy to help!
```
Comment (debdutdeb) [inline README.md:71]:
```suggestion
Any other questions, reach out to us at [our website](https://rocket.chat/contact) or you can email us directly at [contact@rocket.chat](mailto:contact@rocket.chat). We’d be happy to help!
```

**Review by debdutdeb — APPROVED** [review]

**Pull request merged**